### PR TITLE
Add WikiANN NER dataset

### PR DESCRIPTION
**Pull request by lewtun**
This PR adds the full set of 176 languages from the balanced train/dev/test splits of WikiANN / PAN-X from: https://github.com/afshinrahimi/mmner

Until now, only 40 of these languages were available in `datasets` as part of the XTREME benchmark

Courtesy of the dataset author, we can now download this dataset from a Dropbox URL without needing a manual download anymore 🥳, so at some point it would be worth updating the PAN-X subset of XTREME as well 😄 

Link to gist with some snippets for producing dummy data: https://gist.github.com/lewtun/5b93294ab6dbcf59d1493dbe2cfd6bb9

P.S. @yjernite I think I was confused about needing to generate a set of YAML tags per config, so ended up just adding a single one in the README.

### CODE REVIEW

**Comment by lewtun** [comment]
Dataset card added, so ready for review!

File: datasets/wikiann/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+- crowdsourced
+languages: ['ace', 'af', 'als', 'am', 'an', 'ang', 'ar', 'arc', 'arz', 'as', 'ast', 'ay', 'az', 'ba', 'bar', 'bat-smg', 'be', 'be-x-old', 'bg', 'bh', 'bn', 'bo', 'br', 'bs', 'ca', 'cbk-zam', 'cdo', 'ce', 'ceb', 'ckb', 'co', 'crh', 'cs', 'csb', 'cv', 'cy', 'da', 'de', 'diq', 'dv', 'el', 'eml', 'en', 'eo', 'es', 'et', 'eu', 'ext', 'fa', 'fi', 'fiu-vro', 'fo', 'fr', 'frr', 'fur', 'fy', 'ga', 'gan', 'gd', 'gl', 'gn', 'gu', 'hak', 'he', 'hi', 'hr', 'hsb', 'hu', 'hy', 'ia', 'id', 'ig', 'ilo', 'io', 'is', 'it', 'ja', 'jbo', 'jv', 'ka', 'kk', 'km', 'kn', 'ko', 'ksh', 'ku', 'ky', 'la', 'lb', 'li', 'lij', 'lmo', 'ln', 'lt', 'lv', 'map-bms', 'mg', 'mhr', 'mi', 'min', 'mk', 'ml', 'mn', 'mr', 'ms', 'mt', 'mwl', 'my', 'mzn', 'nap', 'nds', 'ne', 'nl', 'nn', 'no', 'nov', 'oc', 'or', 'os', 'pa', 'pdc', 'pl', 'pms', 'pnb', 'ps', 'pt', 'qu', 'rm', 'ro', 'ru', 'rw', 'sa', 'sah', 'scn', 'sco', 'sd', 'sh', 'si', 'simple', 'sk', 'sl', 'so', 'sq', 'sr', 'su', 'sv', 'sw', 'szl', 'ta', 'te', 'tg', 'th', 'tk', 'tl', 'tr', 'tt', 'ug', 'uk', 'ur', 'uz', 'vec', 'vep', 'vi', 'vls', 'vo', 'wa', 'war', 'wuu', 'xmf', 'yi', 'yo', 'zea', 'zh', 'zh-classical', 'zh-min-nan', 'zh-yue']
```
Comment (lhoestq) [inline datasets/wikiann/README.md:6]:
```suggestion
languages: 
- ace:
  - ace
- af:
  - af
- als:
  - als
- am:
  - am
- an:
  - an
- ang:
  - ang
- ar:
  - ar
- arc:
  - arc
- arz:
  - arz
- as:
  - as
- ast:
  - ast
- ay:
  - ay
- az:
  - az
- ba:
  - ba
- bar:
  - bar
- bat-smg:
  - bat-smg
- be:
  - be
- be-x-old:
  - be-x-old
- bg:
  - bg
- bh:
  - bh
- bn:
  - bn
- bo:
  - bo
- br:
  - br
- bs:
  - bs
- ca:
  - ca
- cbk-zam:
  - cbk-zam
- cdo:
  - cdo
- ce:
  - ce
- ceb:
  - ceb
- ckb:
  - ckb
- co:
  - co
- crh:
  - crh
- cs:
  - cs
- csb:
  - csb
- cv:
  - cv
- cy:
  - cy
- da:
  - da
- de:
  - de
- diq:
  - diq
- dv:
  - dv
- el:
  - el
- eml:
  - eml
- en:
  - en
- eo:
  - eo
- es:
  - es
- et:
  - et
- eu:
  - eu
- ext:
  - ext
- fa:
  - fa
- fi:
  - fi
- fiu-vro:
  - fiu-vro
- fo:
  - fo
- fr:
  - fr
- frr:
  - frr
- fur:
  - fur
- fy:
  - fy
- ga:
  - ga
- gan:
  - gan
- gd:
  - gd
- gl:
  - gl
- gn:
  - gn
- gu:
  - gu
- hak:
  - hak
- he:
  - he
- hi:
  - hi
- hr:
  - hr
- hsb:
  - hsb
- hu:
  - hu
- hy:
  - hy
- ia:
  - ia
- id:
  - id
- ig:
  - ig
- ilo:
  - ilo
- io:
  - io
- is:
  - is
- it:
  - it
- ja:
  - ja
- jbo:
  - jbo
- jv:
  - jv
- ka:
  - ka
- kk:
  - kk
- km:
  - km
- kn:
  - kn
- ko:
  - ko
- ksh:
  - ksh
- ku:
  - ku
- ky:
  - ky
- la:
  - la
- lb:
  - lb
- li:
  - li
- lij:
  - lij
- lmo:
  - lmo
- ln:
  - ln
- lt:
  - lt
- lv:
  - lv
- map-bms:
  - other-map-bms
- mg:
  - mg
- mhr:
  - mhr
- mi:
  - mi
- min:
  - min
- mk:
  - mk
- ml:
  - ml
- mn:
  - mn
- mr:
  - mr
- ms:
  - ms
- mt:
  - mt
- mwl:
  - mwl
- my:
  - my
- mzn:
  - mzn
- nap:
  - nap
- nds:
  - nds
- ne:
  - ne
- nl:
  - nl
- nn:
  - nn
- no:
  - no
- nov:
  - nov
- oc:
  - oc
- or:
  - or
- os:
  - os
- pa:
  - pa
- pdc:
  - pdc
- pl:
  - pl
- pms:
  - pms
- pnb:
  - pnb
- ps:
  - ps
- pt:
  - pt
- qu:
  - qu
- rm:
  - rm
- ro:
  - ro
- ru:
  - ru
- rw:
  - rw
- sa:
  - sa
- sah:
  - sah
- scn:
  - scn
- sco:
  - sco
- sd:
  - sd
- sh:
  - sh
- si:
  - si
- simple:
  - simple
- sk:
  - sk
- sl:
  - sl
- so:
  - so
- sq:
  - sq
- sr:
  - sr
- su:
  - su
- sv:
  - sv
- sw:
  - sw
- szl:
  - szl
- ta:
  - ta
- te:
  - te
- tg:
  - tg
- th:
  - th
- tk:
  - tk
- tl:
  - tl
- tr:
  - tr
- tt:
  - tt
- ug:
  - ug
- uk:
  - uk
- ur:
  - ur
- uz:
  - uz
- vec:
  - vec
- vep:
  - vep
- vi:
  - vi
- vls:
  - vls
- vo:
  - vo
- wa:
  - wa
- war:
  - war
- wuu:
  - wuu
- xmf:
  - xmf
- yi:
  - yi
- yo:
  - yo
- zea:
  - zea
- zh:
  - zh
- zh-classical:
  - zh-classical
- zh-min-nan:
  - zh-min-nan
- zh-yue:
  - zh-yue
```
Can you also check that they are valid codes in the tagger ?
For those that are no in the tagger you can just prefix them with `other-`
For example `other-map-bms`

**Review by lhoestq — COMMENTED** [review]
really awesome thank you !
I just left a comment for the language tags

File: datasets/wikiann/README.md (hunk 1 above)
Comment (lhoestq) [reply datasets/wikiann/README.md:6]: you can get the list of valid languages in the tagger here : https://github.com/huggingface/datasets-tagging/blob/main/language_set_full.json

Comment (lewtun) [reply datasets/wikiann/README.md]: Thanks for the tips @lhoestq ! I've fixed the language tags and prefixed 6 that did not belong to the list of valid languages.

Note that to use `no` (Norwegian) as a key, I had to wrap it in quotes to make it valid YAML 

I also added some additional information to the card, but did not have time to fill out all the fields 😞

Comment (lhoestq) [reply datasets/wikiann/README.md]: thanks a lot !

**Review by lhoestq — APPROVED** [review]
looks all good thanks !

**Pull request merged**